website/views/shared/modals/achievements.jade

```diff
@@ -1,99 +1,134 @@
+include ../avatar/generated_avatar
+mixin achievementFooter
+  - var tweet = env.t('achievementShare');
+  .modal-footer(style='margin-top:0', ng-init='loadWidgets()')
+    .container-fluid
+      .row.text-center
+        .col-xs-4
+          a.twitter-share-button(href='https://twitter.com/intent/tweet?text=#{tweet}&via=habitica&url=#{env.BASE_URL}/social/achievement&count=none')=env.t('tweet')
+        .col-xs-4
+          .fb-share-button(data-href='#{env.BASE_URL}/social/achievement', data-layout='button')
+        .col-xs-4
+          a.tumblr-share-button(data-href='#{env.BASE_URL}/social/achievement', data-notes='none')
+
+mixin achievementAvatar(badge,margin)
+  .container-fluid
+    .row
+      .col-xs-4(style='padding:0')
```

Comment:
Neither am I; I commented on that to some extent in the PR description. But so many things needed fiddly little tweaks that it seemed like no matter how I implemented it, it'd add a bunch of ugly clutter somewhere. Suggestions?